src/main/java/com/bootcamp/topic3/cart/CartItem.java

```diff
@@ -0,0 +1,63 @@
+package com.bootcamp.topic3.cart;
+
+public class CartItem {
+	public static int idGenerator = 0;
+	public int id;
+	public String name;
+	public double price;
+	public int quantity;
+
+	public CartItem(String name, double price, int quantitie) {
+
+		this.id = idGenerator++;
```

Comment:
To create an idGenerator I suggest to use [UUID](https://docs.oracle.com/javase/8/docs/api/java/util/UUID.html). Your approcach is not thread safety and less reliable.

`this.id = UUID.randomUUID().toString();`